src/main/java/it/gov/pagopa/pu/workflow/wf/classification/iuf/dto/IufClassificationNotifyPaymentsReportingSignalDTO.java

```diff
@@ -13,11 +13,8 @@
 @Builder
 public class IufClassificationNotifyPaymentsReportingSignalDTO {
   // common fields
-  private Long organizationId;
   private String iuf;
```

Comment:
let's store organizationId here, this way you could read it directly from here instead from the first element of the transfer
It will be the same for the entire list